src/jquery.multiselect.js

```diff
@@ -549,6 +700,34 @@
           checked: nodes.length ? nodes[0].checked : null
         });
       })
+      // collapse button
+      .on('click.multiselect', 'button', function(e) {
```

Comment:
Maybe specify a class for the collapse button so it's more specific to that button instead of any button?

Reply:
@mlh758  Ok.  BTW, are you ok w/ a button vs a styled anchor?

Reply:
Yeah I'm okay with it being a button.